doc_src/language.rst

```diff
@@ -1987,13 +1987,15 @@ You can see the current list of features via ``status features``::
     qmark-noglob            on  3.0 ? no longer globs
     regex-easyesc           on  3.1 string replace -r needs fewer \\'s
     ampersand-nobg-in-token on  3.4 & only backgrounds if followed by a separating character
+    remove-percent-self     off 3.8 %self is no longer expanded (use $fish_pid)
```

Comment:
sounds fine to me. Actually turning it on is a separate decision, possibly not for many years

Reply:
It will be a while, yes, but I would imagine shorter than qmark-noglob.

I expect fewer issues than with stderr-nocaret - that one was bloody everywhere, and it was harder to find than this.

%self was never a huge thing, the main reason why there are still uses is that oh-my-fish is dead more than it being important.